text/Untitled-1

```diff
@@ -0,0 +1,13 @@
+> Only ASCII identifiers are allowed within an external block and in the signature of a function declared `#[no_mangle]`.
```

Comment:
This file shouldn't be checked into the repository (or its history); please amend this commit to delete this file.

Reply:
done